tests/playwright/sanity/container-grid.test.js

```diff
@@ -1,25 +1,16 @@
 const { test, expect } = require( '@playwright/test' );
 const WpAdminPage = require( '../pages/wp-admin-page' );
+const { setExperiment } = require( '../utilities/rest-api' );
 
 test.describe( 'Container Grid tests @container', () => {
-	test.beforeAll( async ( { browser }, testInfo ) => {
-		const context = await browser.newContext();
-		const page = await context.newPage();
-		const wpAdmin = new WpAdminPage( page, testInfo );
-		await wpAdmin.setExperiments( {
-			container: true,
-			container_grid: true,
-		} );
+	test.beforeAll( async () => {
+		await setExperiment( 'container', true );
+		await setExperiment( 'container_grid', true );
```

Comment:
@yotamselementor is there any gain in dropping the object param from setExperiment() and instead calling it multiple times with two arguments?

Reply:
@yotamselementor I agree. It looks like this will be a separate API call for every experiment that we want to activate? Can we just pass an object like we did previously and the backend will activate all of the required experiments with one request?